docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: '3.7'
+
+services:
+
+  # Nginx
+  nginx:
+    image: nginx:alpine
+    container_name: gh-nginx
+    ports:
+      - "80:80"
+    volumes:
+      - .:/symfony
+      - ./docker/nginx.conf:/etc/nginx/conf.d/default.conf
+      - ./docker/logs:/var/log/nginx:cached
+    networks:
+      - gh-network-dev
+
+  # web
+  web:
+    build: ./docker
+    image: gh-web
+    container_name: gh-web
+    working_dir: /symfony
+    ports:
+      - "9003:9001"
+    volumes:
+      - .:/symfony:cached
+      - ./docker/logs:/symfony/var/log:cached
+    networks:
+      - gh-network-dev
+
+  # MariaDB
+  mariadb:
+    image: mariadb:10.2
+    container_name: gh-mariadb
```

Comment:
Are we sure about using MariaDB 10.2 here and not keep using MySQL 5.7, like I assume it should be MySQL 5.7 in production? 

If yes, you will have to update the following file https://github.com/Noelierx/GameHer/blob/master/config/packages/doctrine.yaml#L4-L5

A better way would be to remove `driver` and `server_version` from the configuration file, and change the `DATABASE_URL` env var to `mysql://gameher:gameherpwd@gh-mariadb:3306/gameher?serverVersion=mariadb-10.2.<minor version>`. 
This way you configure `driver`, `server_version` and `url` options at the same time.